File: examples/ruby/DbIpWhoisStatus.rb
# Runnable example: IP WHOIS Snapshot Status (GET /v3.3/status/snapshot/ip/whois)
# Parameters for dbIpWhoisStatus (GET /v3.3/status/snapshot/ip/whois):
#   (no parameters; the API key is set on the client)
require 'whoisfreaks'

WhoisFreaks.configure do |config|
  config.api_key["apiKey"] = "YOUR_API_KEY"   # set once
end

api = WhoisFreaks::DatabasesIPWHOISApi.new
result = api.db_ip_whois_status()
puts result

File: examples/csharp/DbIpWhoisStatus.cs
// Runnable example: IP WHOIS Snapshot Status (GET /v3.3/status/snapshot/ip/whois)
// Parameters for dbIpWhoisStatus (GET /v3.3/status/snapshot/ip/whois):
//   (no parameters; the API key is set on the client)
using System;
using WhoisFreaks.Api;
using WhoisFreaks.Client;

class DbIpWhoisStatus {
    static void Main() {
        var config = new Configuration { BasePath = "https://api.whoisfreaks.com" };
        config.AddApiKey("ApiKeyAuth", "YOUR_API_KEY");  // set once
        var api = new DatabasesIPWHOISApi(config);
        var result = api.DbIpWhoisStatus();
        Console.WriteLine(result);
    }
}
